src/Appwrite/Platform/Tasks/Install.php

```diff
@@ -29,10 +30,11 @@ public function __construct()
             ->param('organization', 'appwrite', new Text(0), 'Docker Registry organization', true)
             ->param('image', 'appwrite', new Text(0), 'Main appwrite docker image', true)
             ->param('interactive', 'Y', new Text(1), 'Run an interactive session', true)
-            ->callback(fn ($httpPort, $httpsPort, $organization, $image, $interactive) => $this->action($httpPort, $httpsPort, $organization, $image, $interactive));
+            ->param('noStart', false, new Boolean(true), 'Run an interactive session', true)
```

Comment:
Should this be camelCase or dash-case?

Reply:
I did camel because `httpPort` and `httpsPort` are camelCase. That said, in CLIs dash-case is more common.

Reply:
Will it be a breaking change if we adapt our style? do we have inconsistencies in other places?

Reply:
Yes, it'll be a breaking change if people are using these scripts and passing the params camelCase. We would have to update utopia-php/cli to support both camelCase and dash-case for backwards compatbility.

This is the only command that has multiple words so it's the only one that has camelCase.

Reply:
Eldad said:

> if we correct the styling and update the docs, the impact should be 0 for current users. Any reasons not to fix it?

@eldadfux, if someone has some script that uses the old syntax, it will break. I do agree the impact is small, though.  That said, I've updated the arguments.